frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -77,8 +77,14 @@ export const insightLogic = kea<insightLogicType<ViewType, FilterType, Timeout>>
             false,
             {
                 // Only show timeout message if timer is still running
-                setShowTimeoutMessage: (_, { showTimeoutMessage }: { showTimeoutMessage: boolean }) =>
-                    showTimeoutMessage,
+                setShowTimeoutMessage: (
+                    _,
+                    {
+                        showTimeoutMessage,
+                    }: {
+                        showTimeoutMessage: boolean
+                    }
```

Comment:
You don't need to add types like this inside reducers. They're automatically applied if you didn't make a typo with the action name.